tox.ini

```diff
@@ -1,49 +1,30 @@
 # The test environment and commands
 [tox]
-envlist = deps, check, test
+envlist = check, test
 skipsdist = True
 
 [testenv:check]
 description = Runs all formatting tools then static analysis (quick)
-deps =
-    --no-deps
-    -r requirements-dev.txt
+deps = flake8
 commands =
     #shed  # combines autoflake, black, isort, and pyupgrade
     flake8
 
-[testenv:test]
+[testenv]
```

Comment:
note to future reviewer: the reason  I removed the `:test` here is because by making the test run the default environment, every other environment inherits it. The practical consequence of this is that you can do `tox -e test-py39` to test against python 3.9, and similar for other versions, without actually having to make environments for every individual python version. Very convenient during development IMO.

One disadvantage is that by declaring it as such, the `check` environment also inherits the settings here, except for those fields that it overrides. The only consequence in this config is that that means that the TERMINUSX_TOKEN is also passed to flake8, who completely ignores it. But something to keep in mind.